modules/documentation/slides_documentation.md

```diff
@@ -0,0 +1,57 @@
+<!-- .slide: data-state="title" -->
+
+# Documentation
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## What is documentation?
+
+- Provides context for your work
+- Allows your collaborators and future you to understand what has been done and why
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## Why document software?
+
+Make your software reusable:
+ 
+- A user should be able to run your software in the same way as you do now 
+- A user should be able to install your software
+- A contributor should be able to add or improve code
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## Documentation types
+
+User documentation
+- Purpose: What does the software do?
+- How can it be used: provide examples
+
+Developer documentation
+- How can your software be modified or extended?
+
+Deployment documentation
+- Hardware and software requirements
```

Comment:
I'm not sure if I agree that 'Deployment documentation' is a separate category of documentation that deserves a place here. I would put it as one of the things that could be in the developer documentation. (but it could also be part of user documentation, for example in the case of NEBULA users actually need to deploy their lessons)

Also, deployment is quite a topic in itself that would need an introduction.

Reply:
Yes, we were actually discussing whether to have a glossary somewhere explaining deployment, but I agree that deployment doc is often part of user or dev and for simplicity we could remove it for now. But I would like to hear @meronvermaas opinion on this.

Reply:
Deployment is also very specific for some projects, I guess for 99% of research software projects it is not relevant because the product is the code itself.

Reply:
Maybe deployment instructions are simply a particular form of installation instructions? For some software that'll be `pip install` and for others it's a bit more involved, but they're both about how to get it to run on your computer.